### PR TITLE
Add -c option to add custom class file

### PR DESCRIPTION
**Pull request by shashwat1998**
For #12 
Add in help options
Add checks for class file
Add class file variable in transformers.py

### CODE REVIEW

**Comment by isabek** [comment]
Good job @shashwat1998. Let's remove cls.txt file. The reason is classes.txt and cls.txt are the same.

**Comment by shashwat1998** [comment]
Just put it there as an example. I'll remove it though!

File: xmltotxt.py [hunk 1]
```diff
@@ -22,10 +23,24 @@ def main():
...
-    transformer = Transformer(xml_dir=xml_dir, out_dir=out_dir)
+    if not os.access(class_file, os.F_OK):
+        print("%s file is missing." % class_file)
+        sys.exit()
+
+    if not os.access(class_file, os.R_OK):
+        print("%s file is not readable." % class_file)
+        sys.exit()
+    
+    if not os.path.splitext(class_file)[-1] == ".txt":
```
Comment (isabek) [inline xmltotxt.py]: We should be able to read files without extensions. Please, get rid of this part of the code.

Comment (shashwat1998) [reply xmltotxt.py]: Sure! Done.

**Comment by isabek** [comment]
Thank you very much @shashwat1998. You did a great job.

**Comment by shashwat1998** [comment]
> Thank you very much @shashwat1998. You did a great job.

Thank you!

**Pull request merged**